contracts/RewardManager.sol

```diff
@@ -141,11 +157,16 @@ contract RewardManager is Ownable, Versionable, Safe {
     external
     onlyHandlers
   {
+    require(
+      rewardProgramIDs.contains(rewardProgramID),
+      "invalid rewardProgramID"
+    );
     rule[rewardProgramID] = blob;
     emit RewardRuleAdded(rewardProgramID, blob);
   }
 
   function lockRewardProgram(address rewardProgramID) external onlyHandlers {
+    rewardProgramIDs.contains(rewardProgramID);
```

Comment:
Can probably remove this line?

Reply:
Good catch, should have been a require, fixed